tests/buildenv.sh

```diff
@@ -2,13 +2,13 @@
 
 set -eu
 
+ARMv7_MAGIC="7f454c4601010100000000000000000002002800"
 ARM_EMU_BIN="${ARM_EMU_BIN:-}"
 BINFMT_MISC="${BINFMT_MISC:-/proc/sys/fs/binfmt_misc/}"
+
 PACKAGES="apk mksquashfs xz"
 FILESYSTEMS="overlay squashfs tmpfs"
 
-ARMv7_MAGIC="7f454c4601010100000000000000000002002800"
-
```

Comment:
:-1: 
why?
ARMv7_MAGIC is an internal non-settable constant, so it is below the 'argument' variables and below the user settable constants.

What IS wrong however is that PACKAGES should be below FILESYSTEMS :p